styleguide/js/zoolander.js

```diff
@@ -139,6 +141,70 @@ jQuery(document).ready(function($) {
   };
 }(jQuery)); //to protect and scope the JQuery alias = $
 
+//Rackspace Zoolander Tabs jQuery Plugin
+(function($){
+  'use strict';
+  $.fn.rsTabs = function(){
+    var $tabs = $(this);
+    var $zoolanderWindow = $(window);
+
+    $tabs.each(function(){
+      var $thisTabSet = $(this);
+      var $tabLink = $thisTabSet.find('.rsTabs-menu .rsTabs-tabItem');
+
+      //function to reset the tabs on page resizes
+      var resizeTimer;
+      $zoolanderWindow.resize(function(){
+        clearTimeout(resizeTimer);
+        resizeTimer = setTimeout(resizeResetTabs, 250);
+      });
+
+      //call back for the tabs reset
+      function resizeResetTabs() {
+        $tabLink.each(function(){
+          var $tL = $(this);
+          if($tL.hasClass('rsTabs-tabActive')){
+            $tL.siblings().removeClass('rsTabs-tabMobile');
+          }
+        });
+      }
+
+      //set first tab active
+      $thisTabSet.find('.rsTabs-menu .rsTabs-tabItem').first().addClass('rsTabs-tabActive');
```

Comment:
Thats a good question. I believe 
```
$thisTabSet.find('.rsTabs-menu > .rsTabs-tabItem')
```
would be the quicker option, once jquery finds .rsTabs-menu it then searches for .rsTabs-tabItem inside the menu only.

This option
```
$thisTabSet.find('.rsTabs-tabItem')
```
Will search the entire tabs for .rsTabs-tabItem, even inside the content area where they aren't there.